modules/check_in/spec/requests/check_in/v2/sessions/appointments_spec.rb

```diff
@@ -212,7 +212,7 @@
           VCR.use_cassette 'check_in/clinics/get_clinics_200' do
             VCR.use_cassette 'check_in/facilities/get_facilities_200' do
               VCR.use_cassette 'check_in/appointments/get_appointments_200' do
-                VCR.use_cassette 'check_in/map/security_token_service_200' do
+                VCR.use_cassette 'map/security_token_service_200_response' do
```

Comment:
although doesn't seem like an issue, I'm just curious why the vcr cassette that is used here was changed? And also, why the cassette `check_in/map/security_token_service_200.yml` was renamed/moved to `map/security_token_service_200_invalid_token.yml`

Reply:
The `check_in/map` cassette directory was a duplicate of the original `map` directory in that its 200 and 401 responses were direct copies of the original 200 and 401 responses. When I rotated the keypair that broke the tests depending on the duplicated responses - rather than start maintaining two identical sets of MAP STS token responses I eliminated the unnecessary duplicate.

Since the original, MAP-created response is now "improperly signed" for the mock environments I re-purposed the old response as the `200_invalid response`, resulting in Github thinking that I was moving a file from `check_in/map` to `map` - I'm not, I just renamed the existing `security_token_service_200_response`.